diff --git a/node_modules/@changesets/cli/dist/cli.cjs.dev.js b/node_modules/@changesets/cli/dist/cli.cjs.dev.js
index 4b68b5d..55e34bc 100644
--- a/node_modules/@changesets/cli/dist/cli.cjs.dev.js
+++ b/node_modules/@changesets/cli/dist/cli.cjs.dev.js
@@ -649,7 +649,7 @@ const getLastJsonObjectFromString = str => {
   return null;
 };
 
-const npmRequestLimit = pLimit__default['default'](40);
+const npmRequestLimit = pLimit__default['default'](3);
 const npmPublishLimit = pLimit__default['default'](10);
 
 function jsonParse(input) {
diff --git a/node_modules/@changesets/cli/dist/cli.cjs.prod.js b/node_modules/@changesets/cli/dist/cli.cjs.prod.js
index 5c4828d..3694ebf 100644
--- a/node_modules/@changesets/cli/dist/cli.cjs.prod.js
+++ b/node_modules/@changesets/cli/dist/cli.cjs.prod.js
@@ -359,7 +359,7 @@ const getLastJsonObjectFromString = str => {
     }
   }
   return null;
-}, npmRequestLimit = pLimit__default.default(40), npmPublishLimit = pLimit__default.default(10);
+}, npmRequestLimit = pLimit__default.default(3), npmPublishLimit = pLimit__default.default(10);
 
 function jsonParse(input) {
   try {
diff --git a/node_modules/@changesets/cli/dist/cli.esm.js b/node_modules/@changesets/cli/dist/cli.esm.js
index d715066..bad525f 100644
--- a/node_modules/@changesets/cli/dist/cli.esm.js
+++ b/node_modules/@changesets/cli/dist/cli.esm.js
@@ -626,7 +626,7 @@ const getLastJsonObjectFromString = str => {
   return null;
 };
 
-const npmRequestLimit = pLimit(40);
+const npmRequestLimit = pLimit(3);
 const npmPublishLimit = pLimit(10);
 
 function jsonParse(input) {
