diff --git a/dist/assemble-release-plan.cjs.dev.js b/dist/assemble-release-plan.cjs.dev.js
index 97fead9facaba8a6ffaa2a2e868173cdfc990116..6f0e2a47086cf09fc5bf853571bef78927eed4a3 100644
--- a/dist/assemble-release-plan.cjs.dev.js
+++ b/dist/assemble-release-plan.cjs.dev.js
@@ -75,7 +75,7 @@ function incrementVersion(release, preInfo) {
     // because semver.inc with prereleases is confusing and this seems easier
 
 
-    version += `-${preInfo.state.tag}.${preVersion}`;
+    version += `-next.${preVersion}`;
   }
 
   return version;
@@ -130,18 +130,9 @@ function determineDependents({
           depType,
           versionRange
         } of dependencyVersionRanges) {
-          if (shouldBumpMajor({
-            dependent,
-            depType,
-            versionRange,
-            releases,
-            nextRelease,
-            preInfo,
-            onlyUpdatePeerDependentsWhenOutOfRange: config.___experimentalUnsafeOptions_WILL_CHANGE_IN_PATCH.onlyUpdatePeerDependentsWhenOutOfRange
-          })) {
-            type = "major";
-          } else {
-            if ( // TODO validate this - I don't think it's right anymore
+          if (['patch', 'minor', 'major'].indexOf(nextRelease.type) > ['patch', 'minor', 'major'].indexOf(type)) {
+            type = nextRelease.type
+          } else if ( // TODO validate this - I don't think it's right anymore
             (!releases.has(dependent) || releases.get(dependent).type === "none") && (config.___experimentalUnsafeOptions_WILL_CHANGE_IN_PATCH.updateInternalDependents === "always" || !semver__default['default'].satisfies(incrementVersion(nextRelease, preInfo), versionRange))) {
               switch (depType) {
                 case "dependencies":
@@ -162,7 +153,7 @@ function determineDependents({
                   }
               }
             }
-          }
+
         }
       }
 
@@ -183,17 +174,23 @@ function determineDependents({
       type,
       pkgJSON
     }) => {
-      // At this point, we know if we are making a change
-      updated = true;
       const existing = releases.get(name); // For things that are being given a major bump, we check if we have already
       // added them here. If we have, we update the existing item instead of pushing it on to search.
       // It is safe to not add it to pkgsToSearch because it should have already been searched at the
       // largest possible bump type.
 
-      if (existing && type === "major" && existing.type !== "major") {
-        existing.type = "major";
-        pkgsToSearch.push(existing);
+      if (existing) {
+        const index = ["patch", "minor", "major"].indexOf(existing.type)
+        if (type === "major") {
+            existing.type = "major";
+        } else if (type === "minor" && index < 1) {
+            existing.type = "minor";
+        } else if (type === "patch" && index < 0) {
+            existing.type = "patch";
+       }
+        // pkgsToSearch.push(existing);
       } else {
+        updated = true;
         let newDependent = {
           name,
           type,
@@ -311,20 +308,15 @@ function getHighestReleaseType(releases) {
   let highestReleaseType = "none";
 
   for (let release of releases) {
-    switch (release.type) {
-      case "major":
+      console.log(release, release.type)
+      if (release.type === "major") {
         return "major";
-
-      case "minor":
+      } else if (release.type === "minor") {
         highestReleaseType = "minor";
-        break;
-
-      case "patch":
+      } else if (release.type === "patch") {
         if (highestReleaseType === "none") {
           highestReleaseType = "patch";
         }
-
-        break;
     }
   }
 
diff --git a/dist/assemble-release-plan.cjs.js b/dist/assemble-release-plan.cjs.js
index f3beb3187f56b512830d4b4ed2724fcdeca8b7d4..42d129091a4a77599e1c6b14ccd5bd8596fbd6c3 100644
--- a/dist/assemble-release-plan.cjs.js
+++ b/dist/assemble-release-plan.cjs.js
@@ -1,7 +1,7 @@
 'use strict';
 
-if (process.env.NODE_ENV === "production") {
-  module.exports = require("./assemble-release-plan.cjs.prod.js");
-} else {
+// if (process.env.NODE_ENV === "production") {
+//   module.exports = require("./assemble-release-plan.cjs.prod.js");
+// } else {
   module.exports = require("./assemble-release-plan.cjs.dev.js");
-}
+// }