diff --git a/node_modules/@envelop/core/typings/plugins/use-masked-errors.d.cts b/node_modules/@envelop/core/typings/plugins/use-masked-errors.d.cts
index 429defd..8474ccf 100644
--- a/node_modules/@envelop/core/typings/plugins/use-masked-errors.d.cts
+++ b/node_modules/@envelop/core/typings/plugins/use-masked-errors.d.cts
@@ -19,4 +19,4 @@ export declare type UseMaskedErrorsOpts = {
     /** The error message that shall be used for masked errors. */
     errorMessage?: string;
 };
-export declare const useMaskedErrors: (opts?: UseMaskedErrorsOpts) => Plugin;
+export declare function useMaskedErrors<PluginContext extends Record<string, any> = {}>(opts?: UseMaskedErrorsOpts): Plugin<PluginContext>;
diff --git a/node_modules/@envelop/core/typings/plugins/use-masked-errors.d.ts b/node_modules/@envelop/core/typings/plugins/use-masked-errors.d.ts
index 429defd..8474ccf 100644
--- a/node_modules/@envelop/core/typings/plugins/use-masked-errors.d.ts
+++ b/node_modules/@envelop/core/typings/plugins/use-masked-errors.d.ts
@@ -19,4 +19,4 @@ export declare type UseMaskedErrorsOpts = {
     /** The error message that shall be used for masked errors. */
     errorMessage?: string;
 };
-export declare const useMaskedErrors: (opts?: UseMaskedErrorsOpts) => Plugin;
+export declare function useMaskedErrors<PluginContext extends Record<string, any> = {}>(opts?: UseMaskedErrorsOpts): Plugin<PluginContext>;
