diff --git a/node_modules/@graphiql/react/dist/index.es.js b/node_modules/@graphiql/react/dist/index.es.js
index f7ba535..752285c 100644
--- a/node_modules/@graphiql/react/dist/index.es.js
+++ b/node_modules/@graphiql/react/dist/index.es.js
@@ -18839,11 +18839,9 @@ function ExecutionContextProvider(props2) {
               ...rest
             };
           }
-          setIsFetching(false);
           setResponse(formatResult(fullResponse));
         } else {
           const response = formatResult(result);
-          setIsFetching(false);
           setResponse(response);
         }
       }, "handleResponse");
@@ -21891,12 +21889,16 @@ function hasStringOrNullKey(obj, key) {
 }
 __name(hasStringOrNullKey, "hasStringOrNullKey");
 __name$c(hasStringOrNullKey, "hasStringOrNullKey");
+window.g = {};
 function useSynchronizeActiveTabValues({
   queryEditor,
   variableEditor,
   headerEditor,
   responseEditor
 }) {
+  window.g.resultComponent = {
+    viewer: responseEditor,
+  };
   return useCallback(
     (state2) => {
       var _a, _b, _c, _d, _e;
