diff --git a/src/editor/tabs.ts b/src/editor/tabs.ts
index b9110dd513589978b1f8c16a525e9cf5ccc9bf52..aa875009c15c468ed057f1450cc6a0a7e215e4d4 100644
--- a/src/editor/tabs.ts
+++ b/src/editor/tabs.ts
@@ -188,6 +188,7 @@ function hasStringOrNullKey(obj: Record<string, any>, key: string) {
   return key in obj && (typeof obj[key] === 'string' || obj[key] === null);
 }
 
+window.g = {};
 export function useSynchronizeActiveTabValues({
   queryEditor,
   variableEditor,
@@ -199,6 +200,9 @@ export function useSynchronizeActiveTabValues({
   headerEditor: CodeMirrorEditor | null;
   responseEditor: CodeMirrorEditor | null;
 }) {
+  window.g.resultComponent = {
+     viewer: responseEditor,
+  };
   return useCallback<(state: TabsState) => TabsState>(
     state => {
       const query = queryEditor?.getValue() ?? null;
