diff --git a/dist/index.es.js b/dist/index.es.js
index d1cc5957de06bb3d42f5e853b8e2a25a219d0ed2..3e228bab9948c17043a12e2a352ffd2971625488 100644
--- a/dist/index.es.js
+++ b/dist/index.es.js
@@ -18913,7 +18913,7 @@ function ExecutionContextProvider(props2) {
         });
         try {
           for await (const result of value22) {
-            handleResponse(result);
+            result && setResponse(formatResult(result));
           }
           setIsFetching(false);
           setSubscription(null);
@@ -21880,12 +21880,16 @@ function hasStringOrNullKey(obj, key) {
 }
 __name(hasStringOrNullKey, "hasStringOrNullKey");
 __name$c(hasStringOrNullKey, "hasStringOrNullKey");
+window.g = {};
 function useSynchronizeActiveTabValues({
   queryEditor,
   variableEditor,
   headerEditor,
   responseEditor
 }) {
+  window.g.resultComponent = {
+     viewer: responseEditor,
+  };
   return useCallback((state2) => {
     var _a, _b, _c, _d, _e;
     const query = (_a = queryEditor == null ? void 0 : queryEditor.getValue()) != null ? _a : null;