diff --git a/node_modules/@guild-docs/client/docs/index.d.ts b/node_modules/@guild-docs/client/docs/index.d.ts
index 7b4804a..0823538 100644
--- a/node_modules/@guild-docs/client/docs/index.d.ts
+++ b/node_modules/@guild-docs/client/docs/index.d.ts
@@ -1,9 +1,9 @@
-import type { FC } from 'react';
+import type { FC, Element } from 'react';
 export declare const DocsContainer: import("@chakra-ui/react").ChakraComponent<"section", {}>;
 export declare const DocsNavigation: import("@chakra-ui/react").ChakraComponent<"aside", {}>;
 export declare const DocsNavigationDesktop: import("@chakra-ui/react").ChakraComponent<"div", {}>;
 export declare const DocsNavigationMobile: import("@chakra-ui/react").ChakraComponent<"div", {}>;
 export declare const DocsTitle: import("@chakra-ui/react").ChakraComponent<"h2", {}>;
-export declare const DocsTOC: FC;
+export declare const DocsTOC: FC<{ children: Element[] }>;
 export declare const DocsContent: import("@chakra-ui/react").ChakraComponent<"article", {}>;
-export type {} from '@chakra-ui/system';
+export type { } from '@chakra-ui/system';
