diff --git a/src/core/sync/write_types/index.js b/src/core/sync/write_types/index.js
index de91ace4d2e074cb519972c86d05af4825c6631a..1f7553ac1cd68e5f12ef07d76c5de0882a33fc60 100644
--- a/src/core/sync/write_types/index.js
+++ b/src/core/sync/write_types/index.js
@@ -617,7 +617,9 @@ export function tweak_types(content, is_server) {
 					}
 				}
 			}
-			modified ||= _modified;
+			if(modified){
+				modified = _modified;
+			}
 			return _modified;
 		}
 