diff --git a/index.d.ts b/index.d.ts
index 2da5af196e3c1ab36820f0d595f5640d3d947476..4aa67543be3997a4adba73fcfeefac5af0d7296e 100755
--- a/index.d.ts
+++ b/index.d.ts
@@ -5,7 +5,7 @@
 
 import RealIORedis = require('ioredis');
 
-export default RealIORedis;
+export default RealIORedis.default;
 export type AddressFromResponse = RealIORedis.AddressFromResponse;
 export type BooleanResponse = RealIORedis.BooleanResponse;
 export type Callback<T> = RealIORedis.Callback<T>;