diff --git a/node_modules/fastify/fastify.js b/node_modules/fastify/fastify.js
index 8ac6f73..39f6e9f 100644
--- a/node_modules/fastify/fastify.js
+++ b/node_modules/fastify/fastify.js
@@ -1,5 +1,15 @@
 'use strict'
 
+// Patch for Node 12
+// TODO: Remove this after we drop Node 12 support
+if (!globalThis.FinalizationRegistry) {
+  globalThis.FinalizationRegistry = function () {
+    return {
+      register() {}
+    }
+  }
+}
+
 const VERSION = '4.0.2'
 
 const Avvio = require('avvio')
