diff --git a/node_modules/graphiql-explorer/dist/Explorer.js b/node_modules/graphiql-explorer/dist/Explorer.js
index 3bef05d..77e754d 100644
--- a/node_modules/graphiql-explorer/dist/Explorer.js
+++ b/node_modules/graphiql-explorer/dist/Explorer.js
@@ -2631,42 +2631,48 @@ var ExplorerWrapper = function (_React$PureComponent10) {
             flexDirection: 'column',
             overflow: 'hidden'
           } },
-        React.createElement(
-          'div',
-          { className: 'doc-explorer-title-bar' },
-          React.createElement(
-            'div',
-            { className: 'doc-explorer-title' },
-            this.props.title
-          ),
           React.createElement(
-            'div',
-            { className: 'doc-explorer-rhs' },
+            'section',
+            {
+              className: 'doc-explorer'
+            },
+            React.createElement(
+              'div',
+              { className: 'doc-explorer-title-bar' },
+              React.createElement(
+                'div',
+                { className: 'doc-explorer-title' },
+                this.props.title
+              ),
+              React.createElement(
+                'div',
+                { className: 'doc-explorer-rhs' },
+                React.createElement(
+                  'div',
+                  {
+                    className: 'docExplorerHide',
+                    onClick: this.props.onToggleExplorer },
+                  '\u2715'
+                )
+              )
+            ),
             React.createElement(
               'div',
               {
-                className: 'docExplorerHide',
-                onClick: this.props.onToggleExplorer },
-              '\u2715'
+                className: 'doc-explorer-contents',
+                style: {
+                  padding: '0px',
+                  /* Unset overflowY since docExplorerWrap sets it and it'll
+                  cause two scrollbars (one for the container and one for the schema tree) */
+                  overflowY: 'unset'
+                } },
+              React.createElement(
+                ErrorBoundary,
+                null,
+                React.createElement(Explorer, this.props)
+              )
             )
           )
-        ),
-        React.createElement(
-          'div',
-          {
-            className: 'doc-explorer-contents',
-            style: {
-              padding: '0px',
-              /* Unset overflowY since docExplorerWrap sets it and it'll
-              cause two scrollbars (one for the container and one for the schema tree) */
-              overflowY: 'unset'
-            } },
-          React.createElement(
-            ErrorBoundary,
-            null,
-            React.createElement(Explorer, this.props)
-          )
-        )
       );
     }
   }]);
